lerna-debug.log

```diff
@@ -0,0 +1,22 @@
+0 silly argv {
```

Comment:
Please remove log and add `*.log` to [.gitignore](https://github.com/little-spoon-dev/design-system/blob/main/.gitignore):

```sh
git rm lerna-debug.log
echo '*.log' >> .gitignore
```